src/main/java/org/sqlite/jdbc4/JDBC4ResultSet.java

```diff
@@ -310,8 +313,10 @@ public void updateNClob(String columnLabel, Reader reader) throws SQLException {
         throw new SQLFeatureNotSupportedException();
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
why this?

Reply:
Written due to the invocation like Date.getYear(). Already removed.